release-note.html

```diff
@@ -28,17 +28,16 @@
 <body>
 <div id="main-content">
     <div>
-        <h1>What's new in 1.0.0-Beta</h1>
-        <p>This is the beta release of the Choreo Connect 1.0.0.</p>
+        <p>This is the RC1 release of the Choreo Connect 1.0.0.</p>
```

Comment:
Should this be
```suggestion
        <p>Choreo Connect 1.0.0 is Released</p>
```
@praminda

Reply:
+1